README.md

```diff
@@ -30,7 +30,15 @@ You can load a _bundled build_ via
 [unpkg.com](https://unpkg.com/@google/model-viewer/dist/model-viewer.js):
```

Comment:
```suggestion
unpkg.com by including the snippet below. This will automatically load the correct version for the user's browser.
```

Reply:
My thought was this'd make it a little clearer for folks that they should just include the whole snippet, and the browser will choose the proper version?

Reply:
Agree, but it was a little confusing what `This` is referring to in `This will automatically load...`. Maybe use `The snippet will automatically load...`?

Reply:
Yeah, that sounds good to me.